src/main/java/seedu/address/logic/Messages.java

```diff
@@ -13,6 +13,10 @@
 public class Messages {
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
+    public static final String MESSAGE_UNKNOWN_COMMAND_CLEAR_CONFIRMATION =
+            "Unknown command, enter Y to clear or N to abort";
+    public static final String MESSAGE_UNKNOWN_COMMAND_DELETE_CONFIRMATION =
+            "Unknown command, enter Y to delete or N to abort";
```

Comment:
These two confirmation messages are very similar. Would it be better to have one single message ```Unknown command, enter Y to %s or N to abort```, and use String.format later on?